packages/frontend/amp/components/Standfirst.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import { headline } from '@guardian/pasteup/typography';
+import { headline, textSans } from '@guardian/pasteup/typography';
 import { css, cx } from 'emotion';
 import { palette } from '@guardian/pasteup/palette';
 import { pillarPalette, pillarMap } from '@frontend/lib/pillars';
 
-// TODO - unclear if we need the list styles as well here
+// listStyles are used in paid content standfirsts, with a fallback for normal pillars
 const listStyles = (pillar: Pillar) => css`
     li {
         margin-bottom: 6px;
         padding-left: 20px;
-        ${headline(2)};
+        ${pillar === 'labs' ? textSans(7) : headline(2)};
+        ${pillar === 'labs' && 'font-weight: 700;'}
```

Comment:
At the least it is probably worth adding vertical whitespace above and below these lines so they are visually more obvious. But it is probably even better to create a function for this part so that the logic is removed from the css declaration itself. You could even do this using `cx` on the className part so that the CSS itself is kept as dumb as possible?

Reply:
Yes, I was reflecting whether separating out all the paidContent CSS into a composable class would be a better way to go. Sounds like you agree so will go this route!